README.md

```diff
@@ -134,10 +65,18 @@ Feel free to create a GitHub issue or join the Bitbucket Plugin channel on our c
 
 Bitbucket user tokens are AES-encrypted with an At Rest Encryption Key configured in the plugin's settings page. Once encrypted, the tokens are saved in the `PluginKeyValueStore` table in your Mattermost database.
 
-## Development
+## Contribute
+
+### Development
 
 This plugin contains both a server and web app portion. Read our documentation about the [Developer Workflow](https://developers.mattermost.com/extend/plugins/developer-workflow/) and [Developer Setup](https://developers.mattermost.com/extend/plugins/developer-setup/) for more information about developing and extending plugins.
 
+### Test Your Changes
+
 ## License
 
 This repository is licensed under the [Apache 2.0 License](https://github.com/mattermost/mattermost-plugin-bitbucket/blob/master/LICENSE).
+
+## Security Vulnerability Disclosure
+
+## Get Help
```

Comment:
Is it expected that these sections are empty?

Reply:
@hanzei in the ticket I was asked to add sections even if there is no text from the previous README to fill them.

Reply:
@kat-kan - I can appreciate the misunderstanding here. While the Help Wanted says "Add section headings if they’re applicable but missing", the request is to add headings if it makes sense to do so.

In cases where it doesn't make sense (because that section doesn't have any content), we won't want an empty heading. Please remove any empty headings in your PR.